apps/payments/next/jest.config.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+export default {
+  displayName: 'payments-next',
+  preset: '../../../jest.preset.js',
+  transform: {
+    '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '@nx/react/plugins/jest',
+    '^.+\\.[tj]sx?$': ['babel-jest', { presets: ['@nx/next/babel'] }],
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
+  coverageDirectory: '../../../coverage/apps/payments/next',
```

Comment:
When running the tests, with `nx run payments-next:test`, I get the following Validation Warning. Could you please fix the warning?

```
● Validation Warning:

  Unknown option "coverageDirectory" with value "../../../coverage/apps/payments/next" was found.
```

Reply:
Unfortunately, this is a bug that is currently still open in this version of Jest (v29) - [[Bug]: Validation warnings for options like preset or coverage directory](https://github.com/jestjs/jest/issues/13576)